### PR TITLE
add alphafold v3 param parse & config

### PR DESCRIPTION
**Pull request by ZiyaoLi**

### CODE REVIEW

**Review by guolinke — APPROVED** [review]

**Comment by lhatsk** [comment]
AFAICT the parameters are never swapped, isn't there something missing here?

**Comment by guolinke** [comment]
@lhatsk 
image

**Comment by ZiyaoLi** [comment]
@teslacool plz check if current implementation is correct and necessary.

**Comment by lhatsk** [comment]
I was expecting something along this:

```
    LinearSwapParams = lambda l, index: {
        "weights": LinearWeight(torch.cat([l.weight[index:], l.weight[:index]])),
        "bias": LinearBias(torch.cat([l.bias[index:], l.bias[:index]])),
    }
```
Tbf, I also didn't do this and my guess is the network can correct it. Both versions behave very weirdly at the beginning of fine-tuning though. Large initial loss, my guess is I am missing something.

**Comment by teslacool** [comment]
@ZiyaoLi pls review this [PR](https://github.com/dptech-corp/Uni-Fold/pull/125).
